mgmt/include/mgmt/mgmt.h

```diff
@@ -225,18 +227,23 @@ struct mgmt_streamer_cfg {
 struct mgmt_streamer {
     const struct mgmt_streamer_cfg *cfg;
     void *cb_arg;
-    struct cbor_decoder_reader *reader;
-    struct cbor_encoder_writer *writer;
+    struct cbor_nb_reader reader;
+    struct cbor_nb_writer writer;
 };
 
+
+struct buffer_ctxt {
```

Comment:
strictly speaking the docstring is missing

Reply:
and given everything is prefixed with `mgmt_` I would expect it to be called `mgmt_buffer_ctxt`, additionally I am not sure whether it should have the `_ctxt` prefix given it s simply the encode and decode buffers.

Reply:
Fixed

Reply:
@royvandam Could you check my rework and approve?